src/NewSessionView.elm

```diff
@@ -187,65 +185,60 @@ view context model =
                                     [ text (DateUtils.displayDateWithoutTime d) ]
                             )
             in
-                div [ class "form-group" ]
-                    [ div [ class "input-group", onInput UpdateNewSessionDate ]
-                        [ label [ for "day-input" ] [ text "Date " ]
-                        , br [] []
-                        , select [ id "day-input" ]
+                div []
```

Comment:
Also, they don't **do anything** apart from hide the elements they contain. 
This whole part can exist (_and look and act exactly the same_) like this
![screen shot 2017-03-13 at 14 41 53](https://cloud.githubusercontent.com/assets/7805691/23859371/4ab09e3e-07fb-11e7-9ab7-bd74f5c6d358.png)